doc/releases/changelog-0.40.0.md

```diff
@@ -79,17 +79,61 @@
 
 <h4>Bosonic operators 🎈</h4>
 
-* Added `unary_mapping()` function to map `BoseWord` and `BoseSentence` to qubit operators, using unary mapping.
-  [(#6576)](https://github.com/PennyLaneAI/pennylane/pull/6576)
+A new module, :doc:`qml.bose <~.bose>`, for bosonic function is available in PennyLane with the 
+following features.
```

Comment:
This is a great changelog entry, so this is just a very minor comment :) But it would be good for the first line of the entry to be more descriptive of the new feature. E.g., if the reader stopped reading after _the first line_, what is the most important information we would want them to come away with?

E.g.,

> Support for constructing and manipulating Bosonic operators, and converting between Bosonic operators and qubit operators, has been added to PennyLane.

Reply:
Thanks Diego! How's this? 🤔 

```suggestion
A new module called `qml.bose` has been added to PennyLane that includes support for constructing and manipulating Bosonic operators, and converting between Bosonic operators and qubit operators.
```

Reply:
OH MY 🤦 ... @josh146 I thought you were Diego. 🧠 🚮 🤣